### PR TITLE
Fix visibility of participant list singleitem menu

### LINKED ISSUE #3486: Fix three dot menu for participant in participant-list
**Current behaviour:**
The three dot menu is always visible even if no options are available

**Reproduction:**
1. have a group who can see the participants but not manage the participants
2. login as a user from this group
3. navigate to participant-list
4. three-dot menu is visible and clickable but opens empty 

**Wanted behaviour:**
The three dot menu should only be visible if an option is availavle/if the participant can manage participants

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #3486

### CODE REVIEW

**Review by Elblinator — CHANGES_REQUESTED** [review]
A participant with the new Permission can update participants sees too little information in the three dot menu 
What additionally should be visible: 
- edit details
- swap mandates
- send invitation email

**Comment by reiterl** [comment]
This issue is not about, what is seen in the `singleItemMenu`, but about, if the
3 dots icons are shown or not. If the menu is empty, the 3-dots should not be shown.

@Elblinator  please retest, and for the other stuff open a new issue.

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts [hunk 1]
```diff
@@ -454,6 +454,10 @@ export class ParticipantListComponent extends BaseMeetingListViewComponent<ViewU
         await this.repo.removeUsersFromMeeting([user]);
     }
 
+    public canSeeSingleItemMenu(user: ViewUser): boolean {
+        return this.operator.hasPerms(Permission.userCanManage) || this.canChangePassword(user);
```
Comment (Elblinator) [inline client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts:458]: please change to ```this.operator.hasPerms(Permission.userCanUpdate)```

**Review by Elblinator — CHANGES_REQUESTED** [review]
Please change the permission

I'll create a new Issue for my previous CR

File: client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts (hunk 1 above)
Comment (reiterl) [reply client/src/app/site/pages/meetings/pages/participants/pages/participant-list/components/participant-list/participant-list.component.ts]: done

**Comment by reiterl** [comment]
okay, I  have changed the permission check in canSeeSingleItemMenu.

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]

**Pull request merged**